docs/common-coreboot-docs/vboot_signing.md

```diff
@@ -0,0 +1,50 @@
+# coreboot vboot signing
+
+## Introduction
+
+Verified Boot is a method of verifying that the firmware compents haven't been
+tampered with. It uses cryptographic signatures to determine whether the
+firmware comes from trusted source. This document covers the procedure for
+generating vboot keys and configuring the coreboot build system to sign the
+binaries with the generated keys.
+
+## Preparation
+
+Navigate to the coreboot tree.
+
+## Generating keys
+
+Start by updating git submodules:
+
+        git submodule update --init --checkout
```

Comment:
No code blocks?

Reply:
Double indent is also displayed in a code block, but i can change it to ``` for clarity

Reply:
Updated